CONTRIBUTING_DOCS.md

```diff
@@ -204,11 +204,21 @@ An anchor link for each heading is automatically created. It consists of the tit
 
 Because the documentation site includes content for past versions as well as the latest version, it contains many duplicate pages. When Google indexes the site, it needs to know which copy we prefer. We use [jekyll-seo-tag](https://github.com/jekyll/jekyll-seo-tag) to add [canonical URLs](https://support.google.com/webmasters/answer/139066?hl=en) to each page. This helps us to ensure that the latest copy of the page comes up first when people search for information via Google.
 
-Each page should include a `canonical_url` tag that contains the absolute path to the latest copy of the page, even if the latest copy is the page itself. 
+Each page should include a `canonical_url` tag that contains the absolute path to the latest copy of the page, even if the latest copy is the page itself.
 
 You should _not_ need to modify the `canonical_url` metadata unless you are adding, deleting, or renaming a page.
 
+## Code samples
+
+Our site adds a copy button to each code block by default. To ensure that readers can copy and paste the code successfully, follow the [Code samples](https://github.com/projectcalico/calico/blob/contr-update/DOC_STYLE_GUIDE.md#code-samples) recommendations in the DOC_STYLE_GUIDE.
```

Comment:
@emanic this link doesn't seem to exist.

I think you want this: https://github.com/projectcalico/calico/blob/master/DOC_STYLE_GUIDE.md#code-samples